moment/moment.d.ts

```diff
@@ -325,7 +325,7 @@ declare module moment {
       weekdays?: any;
       weekdaysShort?: any;
       weekdaysMin?: any;
-      longDateFormat?: MomentLongDateFormat;
```

Comment:
Isn't this just an additive change (instead of a replacement) and doesn't longDateFormat() return a MomentLongDateFormat?  Such that both lines should exist:

`longDateFormat?: MomentLongDateFormat;`
`longDateFormat(format: string): MomentLongDateFormat;'